miner/README.org

```diff
@@ -31,7 +31,7 @@ process of mathematically "solving" a block. Unless solved, a block cannot be
 included in the chain. Without mining, a blockchain cannot progress and
 transactions will never finalize.
 
-Economically, mining is the principle way to obtain currency on a Chainweb
+Economically, mining is the principal way to obtain currency on a Chainweb
```

Comment:
This is incorrect.  In this case `principle` is the correct spelling.

Reply:
Oops, my bad.  On re-checking this is correct.

Reply:
:gb: